docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -73,8 +73,13 @@ TEKU_DATA_PATH=/home/me/me_node
 data-path: "/home/me/me_node"
 ```
 
-The path to the Teku data directory. The default is the directory in which Teku is installed
-or `/opt/teku/database` if using the Teku Docker image.
+The path to the Teku data directory. The default directory is OS dependant:
+
+* macOS: `~/Library/teku/data`
+* Unix/Linux: `~/.teku/data`
+* Windows: `%localappdata%\teku\data`
```

Comment:
i wouldn't put 'data' on any of these, because the path it's all referring to is just to 'teku' - then under that there's a 'data' folder for database stuff, and a 'log' folder for logs stuff

Reply:
Done